### DISCUSSION TITLE
Multiple farmers

### DISCUSSION THREAD

**Discussion by iWedmak** [Chia-Network/chia-blockchain#4665] [Q&A]
Is it okay to run multiply farmers with same keys? is `Expected time to win` will add?

**Comment by Isanderthul** [comment]
@iWedmak as far as I can understand this does not give any benefit. Your farmer is checking the blockchain, checking it twice does not help.

**Reply by iWedmak** [reply]
sooo you saying that even if plots are different on each farmer they will not add my changes to win?

**Reply by keeperlink** [reply]
Every UNIQUE plot give you a chance to win no matter on how you split it between farmers
